.github/workflows/test.yml

```diff
@@ -63,6 +63,7 @@ jobs:
       uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.python }}
+        allow-prereleases: true
```

Comment:
Is this needed? Python 3.13 should not be a prerelease anymore?

Reply:
@The-Compiler to Python 3.13 no, Github already release the final version, but it could be useful if we want to prepare the ci/code to next version!

In the manifest 3.14 is now available https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json.

Do you think it's better to take it off?

Reply:
I think it is published now, so we can take it off? I'm doing it to see.